v22.1/vault-db-secrets-tutorial.md

```diff
@@ -0,0 +1,440 @@
+
+1. Authenticate to Vault as the CockroachDB user.
+
+    Use the client token to authenticate to Vault with limited permissions.
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    vault login $VAULT_CLIENT_TOKEN
+    ~~~
+    ~~~txt
+    Success! You are now authenticated. The token information displayed below
+    is already stored in the token helper. You do NOT need to run "vault login"
+    again. Future Vault requests will automatically use this token.
+    ~~~
+
+1. Confirm the limited permissions of the role.
+
+    To confirm that you have assumed a role with limited permissions, try listing the current leases on the `crdb-role`, as we did [previously](#). You should see a permissions error.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    vault list sys/leases/lookup/database/creds/crdb-role/
+    ~~~
+    ~~~txt
+    Error listing sys/leases/lookup/database/creds/crdb-role: Error making API request.
+
+    Namespace: admin/
+    URL: GET https://roach-test-vault.vault.bfb2290a-670b-4a10-bedf-5aab18e84d69.aws.hashicorp.cloud:8200/v1/sys/leases/lookup/database/creds/crdb-role?list=true
+    Code: 403. Errors:
+
+    * 1 error occurred:
+      * permission denied
+    ~~~
+
+1. Pull the CockroachDB credentials.
+
+    The only thing this policy does have permission to do is pull credentials for the CockroachDB cluster. Let's do that.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    vault read database/creds/crdb-role
+    ~~~
+    ~~~txt
+    Key                Value
+    ---                -----
+    lease_id           database/creds/crdb-role/V3T4UVxeQ9RYsJAk3jZF1Dhl.iMXtW
+    lease_duration     1h
+    lease_renewable    true
+    password           FlOo0p7jMTXjT27hlZZ-H
+    username           v-token-crdb-rol-thfLPlFwex0k9Op0P8qA-1653528652
+    ~~~
+  
+1. Connect to the CockroachDB cluster with your Vault-generated dynamic secret credentials
+
+    1. Using the previous output, add the crdb-role credentials to your environment:
+        {% include_cached copy-clipboard.html %}
+        ~~~shell
+        export USER_NAME=v-token-crdb-rol-thfLPlFwex0k9Op0P8qA-1653528652 # generated CockroachDB client username
+        export PASSWORD=FlOo0p7jMTXjT27hlZZ-H # generated CockroachDB client password
+        export DB_NAME=defaultdb
+        export CLUSTER_NAME=lilac-grizzly-684 # generated CockroachDB client password
+        export HOST=free-tier21.aws-us-west-2.crdb.io
+        export TLS_OPTS="sslrootcert=root.crt&sslmode=verify-full&options=--cluster=${CLUSTER_NAME}"
+        export CLI_DB_CONNECTION_URL="postgresql://$USER_NAME:$PASSWORD@${HOST}:26257/${DB_NAME}?${TLS_OPTS}"
+        ~~~    
+        
+    1. List all the tables in database `defaultdb` to confirm you can connect to your CockroachDB cluster:
```

Comment:
Nit: there's a bit of a mix of colon and period before a command. I prefer colons, for what it's worth. But I think switching them all to one or the other will do fine!